config/boards/shields/the_paintbrush/the_paintbrush_left.keymap

```diff
@@ -22,10 +22,41 @@
 #define KEY_I 5
 #define KEY_O 4
 
+//Copied from the base file for reference only
+#define LAYER_ID_GLOBAL -1
+#define LAYER_ID_BASE 0
+#define LAYER_ID_NUMBERS 1
+#define LAYER_ID_SYMBOLS 2
+#define LAYER_ID_PARENTHETICALS 3
+#define LAYER_ID_NAVIGATION 4
+#define LAYER_ID_BT_SEL 5
+#define LAYER_ID_CUSTOM 6
+#define LAYER_ID_BIG_SYM 7
+#define LAYER_ID_BIG_FUNCTION 8
+
```

Comment:
Can you remove this block?

Reply:
Since these are defined in the base dtsi file, you can't use `LAYER_ID_BASE` in the keymap file without this block

Reply:
Can we isolate them in their own dtsi and then include it in just the user dtsi and in the main dtsi with ifndef's around each?